Assets/MixedRealityToolkit/_Core/Utilities/WebRequestRest/Response.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities.WebRequestRest
```

Comment:
Same again, should be in definitions.  In a subfolder if need be.

Reply:
I don't agree with moving these out of the related namespaces